CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 0.12.11 (28.08.2018)
+
+All issues and pull requests: [0.12.11 milestone](https://github.com/allegro/hermes/milestone/48)
+
+### Features
+
+#### ([899](https://github.com/allegro/hermes/pull/899)) ChronicleMap v3
+
+Starting from this version Hermes will use [ChronicleMap v3](https://github.com/OpenHFT/Chronicle-Map) as a temporary
```

Comment:
Imho this is a good place to write an info about what have changed in configuration. What do you think?